docs/framework/wpf/graphics-multimedia/how-to-create-a-shape-by-using-a-pathgeometry.md

```diff
@@ -21,11 +21,11 @@ A triangle created with a PathGeometry
   
  The previous example showed how to create a relatively simple shape, a triangle. A <xref:System.Windows.Media.PathGeometry> can also be used to create more complex shapes, including arcs and curves. For examples, see [Create an Elliptical Arc](how-to-create-an-elliptical-arc.md), [Create a Cubic Bezier Curve](how-to-create-a-cubic-bezier-curve.md), and [Create a Quadratic Bezier Curve](how-to-create-a-quadratic-bezier-curve.md).  
   
- This example is part of larger sample; for the complete sample, see the [Geometries Sample](https://go.microsoft.com/fwlink/?LinkID=159989).  
+ This example is part of larger sample; for the complete sample, see the [Geometries Sample](https://github.com/Microsoft/WPF-Samples/tree/master/Graphics/Geometery).  
```

Comment:
Can you review this link to go to https://github.com/Microsoft/WPF-Samples/tree/master/Graphics/Geometry everywhere? I just submitted a PR to fix that on that repo

Reply:
ok!

Reply:
just added 3 more files with the same typo!